lib/XML/Feed/Format/MRSS.pm

```diff
@@ -0,0 +1,77 @@
+package XML::Feed::Format::MRSS;
+use strict;
+use warnings;
+
+use base qw( XML::Feed::Format::RSS );
+
+use XML::Feed::Entry::Format::MRSS;
+
+use XML::FeedPP;
+
+sub format { 'MRSS' }
+
+sub identify {
+    my $class   = shift;
+    my $xml     = shift;
+
+    my $feed = XML::FeedPP->new( $$xml );
+    my $is_mrss = 1;
+    if ( index($$xml, "xmlns:media=\"http://search.yahoo.com/mrss/\"" ) < 0 ) {
```

Comment:
instead of setting a bool to true and then setting it to false, just return 0 as soon as you know it's not mrss.

so at line 20 'return 0'